openquake/hazardlib/gsim/akkar_bommer_2010.py

```diff
@@ -317,14 +318,17 @@ def get_mean_and_stddevs(self, sites, rup, dists, imt, stddev_types):
         for spec of input and result values.
         """
 
+        ref_sites = copy.deepcopy(sites)
+        ref_sites.vs30 = 600 * np.ones(len(sites.vs30))
```

Comment:
I don't understand why you ask the user to provide a Vs30 value if then you fix it at 600 m/s. Wouldn't it be easier to simply avoid asking it and warn the user that this GMPE is customised for a specific Vs30 value?